README.md

```diff
@@ -1,80 +1,56 @@
+This application consumes container metrics off the Cloud Foundry Doppler daemon, processes them based on provided metrics template, and then sends them off to a StatsD endpoint.
+CPU, RAM and disk usage metrics for app containers will be sent through to StatsD as a Gauge metric. It will get metrics for all applications provided user has access to. Note that it is still being developed and shouldn't be considered production-ready.
 
 ## Getting Started
 
-* A user who has access to the Cloud Foundry Firehose configured in
-your manifest
-
-```
-properties:
-  uaa:
-    clients:
-      graphite-nozzle:
-        access-token-validity: 1209600
-        authorized-grant-types: authorization_code,client_credentials,refresh_token
-        override: true
-        secret: <password>
-        scope: openid,oauth.approvals,doppler.firehose
-        authorities: oauth.login,doppler.firehose
-
-```
-
-* A Graphite and StatsD server (see [here](https://github.com/CloudCredo/graphite-statsd-boshrelease) for a Graphite/StatsD BOSH release).
-* Golang installed and configured (see [here](https://golang.org/doc/install) for a tutorial on how to do this).
-* godep (see [here](https://github.com/tools/godep) for installation instructions).
-
-Once you've met all the prerequisites, you'll need to download the library and install the dependencies:
-
-```
-mkdir -p $GOPATH/src/github.com/pivotal-cf
-cd $GOPATH/src/github.com/pivotal-cf
-git clone git@github.com:pivotal-cf/graphite-nozzle.git
-cd graphite-nozzle
-godep restore
-godep go build
-```
-
-Finally, run the app:
-
+* Clone this repository
+* Deploy the app to CloudFoundry.You can use application flags or envirnoment variables to configure it.
+  * Flags:
+    * `--api-endpoint` - CloudFoundry API endpoint
+    * `--statsd-endpoint` - Statsd endpoint
+    * `--username` - UAA username
+    * `--password` - UAA password
+    * `--skip-ssl-validation` - Do not validate SSL certificate
+    * `--debug` - Enable debug mode. This disables forwarding to statsd and prints to stdout
+    * `--update-frequency` - The time in seconds, that takes between each apps update call.
+    * `--metric-template` - The template that will form a new metric namespace. This uses [Go lang templating engine](https://golang.org/pkg/text/template/)
+
+  * Example manifest:
 ```
-./graphite-nozzle --help
+---
+applications:
+- name: metric-exporter
+  memory: 100M
+  instances: 1
+  buildpack: go_buildpack
+  health-check-type: none
+  no-route: true
+  env:
+    API_ENDPOINT: https://api.10.244.0.34.xip.io
+    STATSD_ENDPOINT: 10.244.11.2:8125
+    STATSD_PREFIX: myproject
+    USERNAME: cloud_foundry_user
+    PASSWORD: cloud_foundry_password
+    SKIP_SSL_VALIDATION: false
+    DEBUG: false
+    UPDATE_FREQUENCY: 300
+    METRIC_TEMPLATE: {{.Space}}.{{.App}}.{{.Metric}}
 ```
 
```

Comment:
Shall we explain all  the possible behaviours of this application. Like using a "subscriptionID" and not passing the aapGUID to get the global CF events. See https://github.com/alphagov/paas-cf-apps-statsd/pull/1/commits/c441a3b84fa15394dcab96f340d50e35248ad809#diff-7ddfb3e035b42cd70649cc33393fe32cR83